client/james.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>James Todd</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+        integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+    <link rel="stylesheet" href="../client/assets/css/james.css">
```

Comment:
This href will need to be adjusted to reflect the server's static root. In a production environment, this will usually not be relative to the directory it's in during development. If you look at my `karsten.html` file as an example, you'll notice my path is `/css/karsten.css` which means that the `assets folder is the root for static assets`

Reply:
You can also see this in `server.js`:

```
// Tell node where to serve static files from
app.use(express.static(path.join(__dirname, '../client/assets')));
```